example_test.go

```diff
@@ -0,0 +1,47 @@
+package gobrake_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+
+	"github.com/airbrake/gobrake"
+)
+
+func ExampleNewBlacklistKeysFilter() {
+	notifier := gobrake.NewNotifier(1, "key")
+	filter := gobrake.NewBlacklistKeysFilter("X-Real-Ip", "hello[0-9]+")
```

Comment:
Remove `hello[0-9]+`. This should show how to filter sensitive data like password and secret.